test/integration/prod_server/controls/vault_spec.rb

```diff
@@ -1,25 +1,45 @@
 # frozen_string_literal: true
 
+config_json, service_cmd, path_to_etc_vault =
+  case system.platform[:family]
+  when 'bsd'
+    %w[/usr/local/etc/vault/conf.d/config.json
+       service\ vault\ status
+       /usr/local/etc/vault]
+  else
+    %w[/etc/vault/conf.d/config.json
+       journalctl\ -u\ vault
+       /etc/vault]
+  end
+
 describe command('/usr/local/bin/vault -version') do
   its(:exit_status) { should eq 0 }
   its(:stderr) { should be_empty }
   # https://rubular.com/r/vVeCVuHAmtTYt3
   its(:stdout) { should match(/^Vault v[0-9.]+ \('[0-9a-f]+'\)/) }
 end
 
-describe command('getcap $(readlink -f /usr/local/bin/vault)') do
-  its(:exit_status) { should eq 0 }
-  its(:stderr) { should be_empty }
-  # https://rubular.com/r/JApIMY1oNqGRZ8
-  its(:stdout) { should match(%r{/vault\s?=? cap_ipc_lock[+=]ep$}) }
+control 'TODO: ...' do
+  title 'TODO: ...'
```

Comment:
Same issue as the [previous comment](https://github.com/saltstack-formulas/vault-formula/pull/54/files#r604397682).